packages/office-ui-fabric-react/src/components/Slider/Slider.styles.ts

```diff
@@ -33,7 +34,7 @@ export const getStyles = (props: ISliderStyleProps): ISliderStyles => {
   };
 
   const slideBoxInactiveSectionStyles = !props.disabled && {
-    backgroundColor: theme.palette.themeLight,
+    backgroundColor: theme.palette.themeLight, // There's no semantic slot with a value of hemeLight in the noneVariant, only strong & soft
```

Comment:
1) Is this the right place to document these choices? Maybe.
2) The logic is backwards. It should not be "I need the themeLight slot, what slot has that color?" But rather "what semantic slot would make sense here?" If the answer is "none" then maybe we need to add a new semantic slot.
3) themeLight is spelled wrong.